microprofile/testing/mock-beans/pom.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    Copyright (c) 2024 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.helidon.microprofile.testing</groupId>
+        <artifactId>helidon-microprofile-testing-project</artifactId>
+        <version>4.0.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>helidon-microprofile-testing-mock-beans</artifactId>
+    <name>Helidon Microprofile Testing Mock Beans</name>
+
+    <description>
+        Integration with Mock Beans to support tests with CDI injection
+    </description>
+
+    <dependencies>
+        <dependency>
+            <groupId>jakarta.enterprise</groupId>
+            <artifactId>jakarta.enterprise.cdi-api</artifactId>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
+        </dependency>
```

Comment:
We still expect direct mockito usage, so having it as provided can work. However we do still want to manage the version for convenience.

Reply:
Making this a `provided` dependency would not be the ideal solution I think. Customer can still version managed `mockito-core` to any compatible version they need.
My reasoning - Junit and Helidon are expected dependencies, as you are writing tests for a Helidon application.
Mockito is not "auto-include" for tests. 

So we can use `provided`, but it may confuse the users, as the feature would fail to compile unless they explicitly add `mockito` to their project as well.

Reply:
I'm assuming that when using `@MockBean` users also always use Mockito explicitly.

E.g.
```java
import io.helidon.microprofile.testing.junit5.HelidonTest;
import io.helidon.microprofile.testing.mockbeans.MockBean;

import org.junit.jupiter.api.Test;
import org.mockito.Mockito;

@HelidonTest
class MyTest {

    @MockBean
    private MyService service;

    @Inject
    private WebTarget target;

    @Test
    void testMyService() {
        Mockito.when(service.test()).thenReturn("Mocked");
        response = target.path("/test").request().get(String.class);
        assertThat(response, is("Mocked"));
    }
}
```

One may not actually not use Mockito directly (even though it doesn't make sense) in which case you'd have a runtime error. This is probably a good reason not to use `provided`.

Otherwise, users would have to resolve the `org.mockito.Mockito` import and would naturally add the mockito Maven dependency.

Reply:
To me it looks equally confusing that Helidon will not bring automatically junit or mockito as a transitive dependency. So it is consistent that if junit is not coming as a transitive dependency, then mockito is also not coming as a transitive dependency.

I am setting it as provided.